bin/pt-online-schema-change

```diff
@@ -10056,7 +10056,7 @@ sub main {
       }
    }
 
-   if ($vp->cmp('8.0') > -1 && $vp->flavor() !~ m/maria/i && $alter_fk_method eq 'drop_swap') {
+   if (($vp->cmp('8.0') >= 0 && $vp->cmp('8.0.14') <= 0) && $vp->flavor() !~ m/maria/i && $alter_fk_method eq 'drop_swap') {
```

Comment:
The goal is here to only throw an error between 8.0 and 8.0.14. 

I‘m not sure if the usage of cmp() is correct.

Reply:
seems good to me

Reply:
shall I change it?

Reply:
I don't think 1 version off is so bad. Kindof a  little margin? lol

Reply:
yeah it's one version off into the right direction (8.0.15) and not 8.0.13 right?

Reply:
I infer so.